docs/user-guide/cli-using-usingcli.md

```diff
@@ -1,32 +1,40 @@
 # Using Zowe CLI
 
-In this section, learn about how to use Zowe CLI, including connecting to the mainframe, managing profiles, integrating with API Mediation Layer, and more.
+In this section, learn how to use Zowe CLI, including how to connect to the mainframe, manage profiles, integrate with the API Mediation Layer, and more.
 
 You can use the CLI interactively from a command window on any computer on which it is installed, or run it in a container or automation environment.
 
-**Tip:** If you want to use the CLI together with a screen reader to provide accessibility, we recommend using the Mac™ Terminal application enabled for Accessibility through [System Preferences > Accessibility](https://support.apple.com/zh-sg/guide/terminal/trml1020/mac). On Windows™, adjust the Properties settings in Command Prompt. For other operating systems, or for alternative terminals, check the specification for the terminal to ensure that it meets accessibility requirements.
+:::tip
 
-## Supported CPU architectures, operating systems, and package/resource managers
+If you want to use the CLI together with a screen reader to provide accessibility, we recommend using the Mac™ Terminal application enabled for Accessibility through [System Preferences > Accessibility](https://support.apple.com/zh-sg/guide/terminal/trml1020/mac). On Windows™, adjust the Properties settings in Command Prompt. For other operating systems, or for alternative terminals, check the specification for the terminal to ensure that it meets accessibility requirements.
```

Comment:
I'm not sure that this paragraph provides much value.

Essentially, the guidance is for users to review the accessibility settings for their OS or terminal. It also sounds like you can only use a screen reader on macOS, but Windows also provides the same functionality.